javav2/usecases/creating_redshift_application/Readme.md

```diff
@@ -0,0 +1,1199 @@
+        $('.xsearch-items').append("<div class='search-item-content'>");
+        $('.xsearch-items').append("<h3 class='search-item-caption'><a href='#'>"+title+"</a></h3>");
+        $('.xsearch-items').append("<className='search-item-meta mb-15'>");
+        $('.xsearch-items').append("<className='list-inline'>");
+        $('.xsearch-items').append("<p><b>"+date+"</b></p>");
+        $('.xsearch-items').append("<p><b>'Posted by "+author+"</b></p>");
+        $('.xsearch-items').append("<div>");
+        $('.xsearch-items').append("<h6>"+body +"</h6>");
+        $('.xsearch-items').append("</div>");
+       });
+      }
+
+## Create a JAR file for the application
+
+Package up the project into a .jar (JAR) file that you can deploy to Elastic Beanstalk by using the following Maven command.
+
+	mvn package
+
+The JAR file is located in the target folder.
+
+![AWS Tracking Application](images/Target.png)
+
+The POM file contains the **spring-boot-maven-plugin** that builds an executable JAR file that includes the dependencies. Without the dependencies, the application does not run on Elastic Beanstalk. For more information, see [Spring Boot Maven Plugin](https://www.baeldung.com/executable-jar-with-maven).
+
+## Deploy the application to Elastic Beanstalk
+
+Sign in to the AWS Management Console, and then open the Elastic Beanstalk console. An application is the top-level container in Elastic Beanstalk that contains one or more application environments (for example prod, qa, and dev, or prod-web, prod-worker, qa-web, qa-worker).
+
+If this is your first time accessing this service, you will see a **Welcome to AWS Elastic Beanstalk** page. Otherwise, you’ll see the Elastic Beanstalk Dashboard, which lists all of your applications.
+
+#### To deploy the application to Elastic Beanstalk
+
+1. Open the Elastic Beanstalk console at https://console.aws.amazon.com/elasticbeanstalk/home.
+2. In the navigation pane, choose  **Applications**, and then choose **Create a new application**. This opens a wizard that creates your application and launches an appropriate environment.
+3. On the **Create New Application** page, enter the following values:
+   + **Application Name** - Redshift App
+   + **Description** - A description for the application
+4. Choose **Create**.
+5. Choose **Create a new environment**.
+6. Choose **Web server environment**.
+7. Choose **Select**.
+8. In the **Environment information** section, leave the default values.
+9. In the **Platform** section, choose **Managed platform**.
+10. For **Platform**, choose **Java** (accept the default values for the other fields).
+11. In the **Application code** section, choose **Upload your code**.
+12. Choose **Local file**, and then select **Choose file**. Browse to the JAR file that you created.  
+13. Choose **Create environment**. You'll see the application being created. When you’re done, you will see the application state the **Health** is **Ok** .
```

Comment:
-> the application state Health is Ok.
(remove 'the')

Reply:
fixed